pkg/provisioner/provisioner.go

```diff
@@ -95,4 +95,7 @@ const (
 	// StateDeprovisioning means we are removing an image from the
 	// host's disk(s)
 	StateDeprovisioning = "deprovisioning"
+	// StateInspecting means we are running the agent on the host to
+	// learn about the hardware components available there
+	StateInspecting = "inspecting"
```

Comment:
Not an issue with this patch per se, but shouldn't these have a type?

Reply:
Yes, good point. I'll do that after this PR merges so I can get them all at once.